SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialDispatcher.cpp

```diff
@@ -79,6 +79,7 @@ void USpatialDispatcher::ProcessOps(Worker_OpList* OpList)
 			break;
 		case WORKER_OP_TYPE_CREATE_ENTITY_RESPONSE:
 			Receiver->OnCreateEntityResponse(Op->create_entity_response);
+			OnCreateEntityResponse(Op->create_entity_response);
```

Comment:
Like I stated above, This should be moved into the Receiver and we should keep the Dispatcher to be minimal.